scripts/nemo/qnn/nemotron-speech-streaming-en-0.6b/run.sh

```
@@ -0,0 +1,29 @@
+chunk_size_ms=1120
+if [ $# -ge 1 ]; then
+  chunk_size_ms="$1"
+fi
+
+
+model_id=nemotron-speech-streaming-en-0.6b
+if [ $# -ge 2 ]; then
+  model_id="$2"
+fi
+
+python3 ./wrapper.py --chunk-size-ms $chunk_size_ms --model-id $model_id
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)